pydata_center/monitoring/templates/base.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+
+    <title>{% block title %}{% endblock %}</title>

```

Comment:
Add a fallback title. So the page doesn’t show a blank title when a child template forgets to define one:
```
<title>{% block title %}Default Title{% endblock %}</title>
```